.github/workflows/tox.yml

```diff
@@ -0,0 +1,74 @@
+# yamllint disable rule:line-length
+name: tox
+on:  # yamllint disable-line rule:truthy
+  - pull_request
+jobs:
+  python:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        pyver: ['2.7', '3.6', '3.7', '3.8']
+    steps:
+      - name: checkout PR
+        uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.pyver }}
+      - name: Install platform dependencies
+        run: |
+          set -euxo pipefail
+          .travis/preinstall
+      - name: Install pip, tox, python dependencies
+        run: |
+          set -euxo pipefail
+          python -m pip install --upgrade pip
+          pip install tox
```

Comment:
Could this be moved in the preinstall script?

Reply:
See above.